src/platform/forms-system/src/js/review/ReviewPage.jsx

```diff
@@ -37,6 +37,7 @@ class ReviewPage extends React.Component {
           formConfig={formConfig}
           pageList={pageList}
           path={path}
+          renderErrorMessage
```

Comment:
This makes the `SubmitController` in `ReviewPage` identical to the one in `RoutedSavableReviewPage` - the only two places where `SubmitController` is used. 

_This is the key part_. If this change causes no negative effects, then the `renderErrorMessage` prop might as well not exist since it will _always_ be true. The rest of the changes are about removing the dead branches.